crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -85,6 +85,25 @@ $env.ENV_CONVERSIONS = {
     }
 }
 
+# Completions distributed by package managers are sourced from a global vendor
+# completions directory If you want to override the default, you can do so like
+# this:
+#
+# $env.NUSHELL_GLOBAL_VENDOR_COMPLETIONS_DIR = ...
+#
+# The default is /usr/share/nushell/vendor_completions.d on Linux and MacOS,
```

Comment:
I'm also not understanding why we have to have the word `vendor` here.

Reply:
We can drop the `vendor_` bit if you want. I just took the name that `fish` uses, because it's a reasonable one IMO, but we don't have to use that.

Reply:
`vendor` has been dropped from the paths.